src/drivers/src/stm32f4xx.c

```diff
@@ -0,0 +1,87 @@
+#include "can_raw.h"
+
+#include "stm32f4xx_hal.h"
+#include <stdint.h>
+
+#define USING_CAN2
+
+static CAN_HandleTypeDef can1;
+static CAN_HandleTypeDef can2;
+static CAN_HandleTypeDef can3;
+
+static void config_handle_parameters(CAN_HandleTypeDef *hcan) {
```

Comment:
These are actually not universal. Prescalers and so on can vary across boards. I think it's better we should assume the peripheral is initialized and have the init function only do canlib specific stuff.